LICENSE-binary

```diff
@@ -576,6 +576,7 @@ See licenses-binary/ for text of these licenses.
     (Apache License, version 2.0) Apache ZooKeeper - Server (org.apache.zookeeper:zookeeper:3.4.6 - https://zookeeper.apache.org/)
     (Apache License, version 2.0) Apache twill (org.apache.twill:* - https://twill.apache.org/)
     (Apache License, version 2.0) Apache tephra (co.cask.tephra:* - https://tephra.apache.org/)
+    (Apache License, version 2.0) Apache Sqoop - Server (org.apache.sqoop:sqoop:1.4.6 - https://github.com/apache/sqoop/)
```

Comment:
LICENSE-sqoop.txt  should be added to  licenses-binary/LICENSE-sqoop.txt 
just as https://github.com/apache/incubator-linkis/pull/2035/files#diff-38e53776e6465b9057e805920c3ec9de556ff1a4595b8c86724845411f91d312